### ISSUE TITLE
Pricing is not correct - region not registering properly?

### ISSUE THREAD

**Issue by moomal** [tfedor/AugmentedSteam#1137]
I live in Pakistan which uses South Asia pricing. The prices are lower than in the US but the currency is still USD. All "best price" and "historical low" prices appear to be from the US store - they are significantly higher than the actual prices I would pay.  I'm using the Chrome extension and checked the settings but couldn't find any way to set the region.  The only option seems to be to manually set the currency, but that doesn't help because as I said, the currency used is USD but the prices are different.

![Screen Shot 2021-04-28 at 7 26 54 AM](https://user-images.githubusercontent.com/5794319/116337177-25eb4800-a7f3-11eb-9768-9fc46dfb81ec.png)

**Comment by candela97** [comment]
yeah if your currency isn't supported by ITAD, the prices can be misleading, see #482
